test/src/components/navigation_bar/navigation_bar_test.dart

```diff
@@ -0,0 +1,482 @@
+import 'package:flutter/gestures.dart';
```

Comment:
Is there any test that specifically tests [UX-1096](https://jira.zebra.com/browse/UX-1096) - something like tapping off center that would have failed before those changes but would now pass?